google/cloud/security/inventory/inventory_loader.py

```diff
@@ -274,6 +286,13 @@ def _send_email(organization_id, cycle_time, cycle_timestamp, status, pipelines,
     except util_errors.EmailSendError:
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
+
+def _configure_logging(configs):
+    """Configures the loglevel for all loggers."""
+    desc = configs.get("loglevel")
+    level = LOGLEVELS[desc]
```

Comment:
I'm not too familiar with how the gflags enum works, but if someone gives the flag value something other than what's found in LOGLEVELS, will we get an error here (because desc isn't found in the LOGLEVELS dict)?